docker/container.go

```diff
@@ -16,69 +17,59 @@ import (
 	"github.com/docker/docker/pkg/term"
 	"github.com/docker/engine-api/client"
 	"github.com/docker/engine-api/types"
-	"github.com/docker/engine-api/types/container"
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/libcompose/config"
 	"github.com/docker/libcompose/labels"
 	"github.com/docker/libcompose/logger"
 	"github.com/docker/libcompose/project"
-	"github.com/docker/libcompose/project/events"
-	util "github.com/docker/libcompose/utils"
 )
 
 // Container holds information about a docker container and the service it is tied on.
 type Container struct {
-	name            string
-	serviceName     string
-	projectName     string
-	containerNumber int
-	oneOff          bool
-	eventNotifier   events.Notifier
-	loggerFactory   logger.Factory
-	client          client.APIClient
-
-	// FIXME(vdemeester) Remove this dependency
-	service *Service
+	// FIXME(vdemeester) Replace with ContainerClient with engine-api vendor update
```

Comment:
The latest `engine-api` split `client.APIClient` into several interface (and use composition for `client.APIClient`). Using `client.ContainerClient` here will :
- make sure we just do container stuff here.
- make it easier to unit test (as we need a smaller *fake* client to implement)